prime-react/src/main/scala/react/primereact/Button.scala

```diff
@@ -24,7 +24,7 @@ import scalajs.js.JSConverters.*
 
 case class Button(
   label:          js.UndefOr[String] = js.undefined,
-  icon:           js.UndefOr[FontAwesomeIcon] = js.undefined,
+  icon:           js.UndefOr[FontAwesomeIcon | String] = js.undefined,
```

Comment:
Are there cases where we actually want to use the `String` version? Should we implement the same pattern in all components accepting `icon` ?

Reply:
It was certainly useful in the demo where we don't have FontAwesome icons available. It also allow using the primeicons, and probably would allow someone to use the FontAwesome webfont icons if they wanted. I think we should probable expand it to other components that accept it.

Reply:
It works, I just have to do `asInstanceOf[String]`. I put a comment at `Message.Severity` to remove the cast if we get rid of ScalablyTyped.